### DISCUSSION TITLE
Restarting timers, rather than continuing from previous session?

### DISCUSSION THREAD

**Discussion by Ander3232** [rcaelers/workrave#332] [Q&A]
Hey guys,

Is this really the current Workrave forum, with only four threads? Anyway...

I'm using Workrave 1.10 in Windows1 10. It's great, but it has an annoying quirk: When I resume from Standby or log back in to Windows, WR continues its timers from wherever they previous were, rather than restarting them. I must then either manually restart them, each time; or dismiss them when they (erroneously) appear, which adds extra time (sometimes, nearly twice as much) to their countdowns.

What I really don't get is, this must be _intended_ (programmed) behaviour. To restart its timers when where they left off, WR must actually record its current timer positions. But why would you ever want it to continuing timing _old_ breaks?

For Standby detection, I assume it could just check the system time every minute or so to see if any significant gaps have occurred, and if so, restart the timers, couldn't it?

For now, is there any workaround for this, or must the code be changed? And if you _did_ intend this behaviour (and again, it seems that you must have), could you please add an option _not_ to do it? Thanks!

**Comment by rcaelers** [comment]
GitHub discussions are just out of beta. I only enabled it a few weeks ago. Workrave also has a mailing list (see https://workrave.org/support/)

When Workrave detects that your computer goes to sleep or standby, it stops all timers. So when your computer resumes from standby, Workrave should reset all timers (assuming the time the computer was in standby/sleep is at least the break duration). I never had any issues with this on my Windows 10 laptop.

Are you really using 1.10? This is a version from 2013 I think. If so, please upgrade to 1.10.48 from https://workrave.org/ and check if that fixes the issue.

**Comment by rcaelers** [comment]
May be related to #314
